package/libs/libunistring/Makefile

```diff
@@ -0,0 +1,64 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libunistring
+PKG_VERSION:=1.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=@GNU/$(PKG_NAME)
+PKG_HASH:=827c1eb9cb6e7c738b171745dac0888aa58c5924df2e59239318383de0729b98
+
+PKG_BUILD_PARALLEL:=1
```

Comment:
this works fine, thank you!

btw, it would be nice to enable parallel compilation for host build too:
`HOST_BUILD_PARALLEL:=1`

Reply:
Does it even work?

Reply:
yeah...the CI failed from another package

adding the configure option `--with-libunistring-prefix` allowed the issue to finally appear on my host machine (I was building it fine before this), the patch resolves it, the combination of the 2 is a solution

Reply:
> Does it even work?

parallel build? seems to work
![image](https://github.com/openwrt/openwrt/assets/22235437/484e4ede-6d3d-47f7-9676-3821a4c6f1fa)

Reply:
Oh yeah I remember why I didn’t add it. It fails to build as parallel sometimes. Maybe at make -j 31.

Reply:
libunistring should be okay (tested with `make package/libs/libunistring/host/{clean,compile} -j31 V=s` for 10 times)

Reply:
oh.... should I leave it then?

by the way I added FPIC for host build, is that appropriate?

Reply:
Fpic is appropriate for static libraries to avoid linking issues.

I forgot what the situation was with macOS. Only static libraries should be built but IIRC macOS really wants libintl to be shared.

Reply:
Is it repeated?
https://github.com/openwrt/packages/tree/master/libs/libunistring